sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/OptimizeInSuite.scala

```diff
@@ -169,24 +169,33 @@ class OptimizeInSuite extends PlanTest {
   }
 
   test("OptimizedIn test: Setting the threshold for turning Set into InSet.") {
-    val plan =
-      testRelation
-        .where(In(UnresolvedAttribute("a"), Seq(Literal(1), Literal(2), Literal(3))))
-        .analyze
-
-    withSQLConf(OPTIMIZER_INSET_CONVERSION_THRESHOLD.key -> "10") {
-      val notOptimizedPlan = OptimizeIn(plan)
-      comparePlans(notOptimizedPlan, plan)
+    {
```

Comment:
Do we need the changes in this file?

Reply:
Yes, it's needed. I added a comment in before the test case to explain the reason. Let me know if it's not clear.